components/datatypes/consent/consent-preferences.schema.json

```diff
@@ -232,6 +232,7 @@
             "inHome",
             "iot",
             "social",
+            "directMail",
```

Comment:
This is a duplicate of phyMail from line 230. While I like the name _directMail_ better, we cannot have both.